This patch is needed if we call autoreconf with autoconf >= 2.70
Patch from https://github.com/curl/curl/commit/f25f602ffd026cc6b5bb0b3a6d8f1abe1bad7881
Fixed in libcurl 7.70.0

--- a/m4/curl-functions.m4
+++ b/m4/curl-functions.m4
@@ -7289,7 +7289,6 @@ dnl CURL_LIBRARY_PATH variable. It keeps the LD_LIBRARY_PATH
 dnl changes contained within this macro.
 
 AC_DEFUN([CURL_RUN_IFELSE], [
-   AC_REQUIRE([AC_RUN_IFELSE])dnl
 
    old=$LD_LIBRARY_PATH
    LD_LIBRARY_PATH=$CURL_LIBRARY_PATH:$old
